tools/kafka/build.gradle

```diff
@@ -0,0 +1,7 @@
+dependencies{
+    implementation misc.lombok
+    implementation cloud.avro
+    implementation cloud.kafkaClients
+    implementation misc.jacksonAnnotations
+    implementation project(':client:cards-client-data')
+}
```

Comment:
Rename directory to tools/kafka-test  (like it is done for example in tools/spring/spring-test-utilities)